vendor/src/github.com/docker/libnetwork/resolver_unix.go

```diff
@@ -0,0 +1,77 @@
+// +build !windows
+
+package libnetwork
+
+import (
+	"fmt"
+	"net"
+	"os"
+	"os/exec"
+	"runtime"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/docker/docker/pkg/reexec"
+	"github.com/docker/libnetwork/iptables"
+	"github.com/vishvananda/netns"
+)
+
+func init() {
+	reexec.Register("setup-resolver", reexecSetupResolver)
```

Comment:
It was added to address https://github.com/docker/libnetwork/issues/1113. But yes, it is not a 1.11 specific issue and it was introduced in 1.10.

@sanimej @mrjana please add details.